src/main/webapp/app/course/plagiarism-cases/instructor-view/plagiarism-cases-instructor-view.component.html

```diff
@@ -49,20 +57,25 @@ <h5 class="mb-0">
                     </div>
                 </div>
                 <div class="card-body">
-                    @for (plagiarismCase of groupedPlagiarismCases[exercise!.id!]; track plagiarismCase) {
+                    @for (plagiarismCase of groupedPlagiarismCases[exercise!.id!]; track plagiarismCase; let j = $index) {
```

Comment:
I would prefer let index = index; so it's clear that it's a index. Since you have multiple indexes you could name them like `plagiarismCaseIndex` or similar